src/RegionsContainer.jsx

```diff
@@ -1,9 +1,14 @@
 import React from 'react';
 
+import { useSelector } from 'react-redux';
+
+import Regions from './Regions';
+
 export default function RegionsContainer() {
+  const { regions } = useSelector((state) => ({
+    regions: state.regions,
+  }));
```

Comment:
```suggestion
  const regions = useSelector((state) => state.regions);
```

단일 상태를 가져올 땐 이렇게 해도 무방합니다.